devnet/README.md

```diff
@@ -0,0 +1,40 @@
+# Memory allocation
+| container          | measured VmHWM | set limit | service/job  |
```

Comment:
While I think this is a great idea,  the error message when we bust a limit is non-existent.  How will we diagnose why the PR is failing?

Reply:
Maybe we could get this to get the error message: 
cmd: `kubectl get pod -o jsonpath='{.items[*].status.containerStatuses[*].lastState.terminated.exitCode},{.items[*].status.containerStatuses[*].lastState.terminated.reason}'` 
output: `137,OOMKilled`

Reply:
added more debugging instructions.